docs/source/_src_user_guide/state_propagation/environment_setup/create_bodies/create_body_settings.rst

```diff
@@ -35,39 +35,56 @@ solar-system bodies. The 'default' settings can be retrieved as follows, using t
 
-where the ``global_frame_origin`` and ``global_frame_orientation`` define the reference frame in which state vectors
-stored in the environment `during` propagation are represented. For more details about reference frames, see
-:ref:`reference_frames`.
+In addition to the above method of creating default bodies, we offer an alternative which is more computationally efficient, at the expense of higher RAM usage and a more limited time interval in which the environment is valid. Such an approach is typically only used when computational speed is very important, and is described in more detail :ref:`here<valid_time_range>`.
 
-In general, it is recommended to choose this as the most 'intuitive' frame origin for your propagation
-(e.g. SSB or Sun for solar system scale propagations, Earth for an Earth orbiter, Mars for a Martian mission, etc.).
+Finally, in case you want to initialize body settings without *any* default settings, the ``body_settings`` in the above script can also be created manually as:
 
-.. warning::
-   This frame definition is *distinct* from the center of propagation that you can define in the propagation setup.
-   For more information, see :ref:`simulation_propagator_setup`.
+    .. tabs::
 
+         .. tab:: Python
 
-The full list of default body settings is available at :ref:`default_environment_models`.
+          .. toggle-header:: 
+             :header: Required **Show/Hide**
 
-.. note::
-   Note that the ephemerides that are used by default are only valid for a somewhat limited time interval
-   (on the order of a century, depending on the specific body).
+             .. literalinclude:: /_src_snippets/simulation/environment_setup/req_create_bodies.py
+                :language: python
 
+          .. literalinclude:: /_src_snippets/simulation/environment_setup/body_list_settings_manual.py
+             :language: python
 
-In addition to the above method of creating default bodies, we offer an alternative which is more computationally efficient, at the expense of higher RAM usage and a more limited time interval in which the environment is valid. Such an approach is typically only used when computational speed is very important, and is described in more detail :ref:`here<valid_time_range>`.
+where the frame origin and orientation have been defined manually as "Earth" and "J2000", respectively.
 
 
 .. _custom_body_settings:
 
 Customization of body settings
-================================
+==============================
 
+Although the default body settings are often very useful, there are various cases where a user will want to override these default settings, or where these default settings cannot be use. The manner in which to overrride the default settings can be divided into three categories:
```

Comment:
The manner in which to ~~overrride~~ *override* the default settings can be divided into three categories: